src/js/components/StarRating/StarRating.js

```diff
@@ -5,9 +5,9 @@ import { StarOutline } from 'grommet-icons/icons/StarOutline';
 import { RadioButtonGroup } from '../RadioButtonGroup';
 
 const StarRating = ({ ...rest }) => {
+  const { value: defaultValue } = rest;
```

Comment:
```suggestion
const StarRating = ({ value: defaultValue, ...rest }) => {
```
Can we change to use this format instead? It is better aligned with the format of the rest of the code base and I think it is slightly easier to read

Reply:
sure i will do it